### PR TITLE
{lib,tools}[GCCcore/12.2.0] subunit v1.4.3, Check v0.15.2, CppUnit v1.15.1

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/a8c793f399a9a6de9966dae3074ba90d for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/0628056fdb35d592f7900af154058a1d for a full test report.

File: easybuild/easyconfigs/s/subunit/subunit-1.4.3-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    ('testtools', '2.6.0', {
+        'checksums': ['28b65e14c0f2d3ecbbfb5f55c9dcde5e4faa80ac16a37a823909a1fe3cbcb30a'],
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages'}
+
+sanity_check_paths = {
+    'files': ['lib/libsubunit.a', 'lib/libsubunit.%s' % SHLIB_EXT],
+    'dirs': [],
```
Comment (bedroge) [inline easybuild/easyconfigs/s/subunit/subunit-1.4.3-GCCcore-12.2.0.eb:55]:
```suggestion
    'dirs': ['include/subunit'],
```

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/443bc5fe2aca79b3ce545b4b496a55fb for a full test report.

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @verdurin!

**Pull request merged**